src/notebook.cc

```diff
@@ -265,14 +256,11 @@ bool Notebook::save_current() {
 }
 
 bool Notebook::close(int page) {
-  JDEBUG("start");
   if (page!=-1) {
     auto view=get_view(page);
     if(view->get_buffer()->get_modified()){
-      if(!save_modified_dialog(page)) {
-        JDEBUG("end false");
+      if(!save_modified_dialog(page))
         return false;
-      }
     }
```

Comment:
I have reviewed, but not tested. Looks good. Only thing I saw was this double if statement.